rpcs3/Emu/RSX/Program/CgBinaryVertexProgram.cpp

```diff
@@ -55,6 +55,32 @@ std::string CgBinaryDisasm::GetDSTDisasm(bool is_sca)
 	std::string ret;
 	std::string mask = GetMaskDisasm(is_sca);
 
+	const std::string_view output_names[] =
+	{
+		"out_diffuse_color",
+		"out_specular_color",
+		"out_back_diffuse_color",
+		"out_back_specular_color",
+		"out_fog",
+		"out_point_size",
+		"out_clip_distance[0]",
+		"out_clip_distance[1]",
+		"out_clip_distance[2]",
+		"out_clip_distance[3]",
+		"out_clip_distance[4]",
+		"out_clip_distance[5]",
+		"out_tc8",
+		"out_tc9",
+		"out_tc0",
+		"out_tc1",
```

Comment:
lol why is this scrambled?

Reply:
It is actually the correct order for some weird reason.